src/utils/export.js

```diff
@@ -46,11 +46,42 @@ export default async (format) => {
         break
     }
     case "txt-clipboard": {
-        // TODO: Read raw save
+        const data = await window.internal.ui.editor.getContent()
+
+        data.shift()
+        let parsedData = ""
+        data.forEach((line) => {
+            const strippedLine = line.replace(/<[^>]*>/g, "")
+            parsedData = `${parsedData}\n${atob(strippedLine)}`
+        })
+        parsedData = parsedData.slice(1)
+
+        console.debug(parsedData)
+
+        navigator.clipboard.writeText(parsedData)
+
         break
     }
     case "txt-file": {
-        // TODO: Read raw save
+        const data = await window.internal.ui.editor.getContent()
+
+        data.shift()
+        let parsedData = ""
+        data.forEach((line) => {
+            const strippedLine = line.replace(/<[^>]*>/g, "")
+            parsedData = `${parsedData}\n${atob(strippedLine)}`
+        })
+        parsedData = parsedData.slice(1)
+
+        console.debug(parsedData)
+
+        const element = document.createElement("a")
```

Comment:
Consider using the `saveAs` function for this.

Reply:
fixed